README.md

```diff
@@ -340,55 +349,55 @@ information, see about_Execution_Policies at http://go.microsoft.com/fwlink/?Lin
 
 ### Known Issues
 
-- The `dsc_log` resource may not appear to work. The ["Log" resource](https://technet.microsoft.com/en-us/library/Dn282117.aspx) writes events to the 'Microsoft-Windows-Desired State Configuration/Analytic' event log, which is [disabled by default](https://technet.microsoft.com/en-us/library/Cc749492.aspx).
+- The `dsc_log` resource might not appear to work. The ["Log" resource](https://technet.microsoft.com/en-us/library/Dn282117.aspx) writes events to the 'Microsoft-Windows-Desired State Configuration/Analytic' event log, which is [disabled by default](https://technet.microsoft.com/en-us/library/Cc749492.aspx).
 
-- You may have issues attempting to use `dsc_ensure => absent` with `dsc_service` with services that are not running.
+- You might have issues if you attempt to use `dsc_ensure => absent` with `dsc_service` with services that are not running.
 
-When setting resources to absent, it is common to specify a minimal statement like so:
+  When setting resources to absent, you might normally specify a minimal statement such as:
 
-~~~
+  ~~~
 dsc_service{'disable_foo':
   dsc_ensure => absent,
   dsc_name => 'foo'
 }
-~~~
+  ~~~
 
-Due to the way the Service DSC Resource sets its defaults, the above minimal statement will report the service as already absent erroneously if the service is currently not running. In order to work around this, you must specify that `State => 'Stopped'` as well as `Ensure => absent'`. The following example will work:
+  However, due to the way the Service DSC Resource sets its defaults, if the service is not currently running, the above statement erroneously reports that the service is already absent. To work around this, specify that `State => 'Stopped'` as well as `Ensure => absent'`. The following example works:
 
-~~~
+  ~~~
```

Comment:
They can be, but they weren't marked that way when they came to me. I can fix this one though.